src/pages/home/report/ReportActionCompose/SuggestionMention.js

```diff
@@ -215,12 +215,12 @@ function SuggestionMention({
 
             // Detect if the last two words contain a mention (two words are needed to detect a mention with a space in it)
             if (lastWord.startsWith('@')) {
-                atSignIndex = leftString.lastIndexOf(lastWord);
+                atSignIndex = leftString.lastIndexOf(lastWord) + afterLastBreakLineIndex;
```

Comment:
@dukenv0307 Won't it have a offset of `1` index because we add `1` to the `afterLastBreakLineIndex`?

Reply:
@dukenv0307 It shouldn't offset `atSignIndex` right? Because it will offset the index by `1` wrongly?

Reply:
Got it. Thank you checking.